themes/digital.gov/layouts/partials/core/get_authors.html

```diff
@@ -1,63 +1,75 @@
+      {{- with $context.Site.GetPage (printf "/%s/%s" "authors" $uid) -}}
+        {{ $displayName := .Params.display_name }}
+        {{ $agency := .Params.agency }}
 
-        <div class="author grid-row">
-          <div class="grid-col-auto">
-            <div class="photo">
-              {{- partial "core/get_author_image.html" (dict "author" $author ) -}}
-            </div>
-          </div>
-          <div class="grid-col-9">
-            <div class="details">
-              <span class="usa-sr-only">
-                Originally posted by
-                {{ $author.Params.display_name | default $uid }} on
-                {{ $.Date.Format "Jan 2, 2006" }}
-              </span>
-              <a
-                href="{{ (printf "authors/%s/" $uid) | absURL }} "
-                title="Posts by {{ $author.Params.display_name | default $uid }}"
-                rel="author"
-                aria-label="Read more articles by {{ $author.Params.display_name | default $uid }}"
-              >
-                {{- $author.Params.display_name | default $uid -}}
-              </a>
-              {{- if $author.Params.agency -}}
-                <p>
-                  {{- $author.Params.agency -}}{{- if $author.Params.location -}}
-                    &nbsp;|&nbsp;{{- $author.Params.location -}}
-                  {{- end -}}
-                </p>
-              {{- end -}}
+        {{ if $isShort }}
+          <a
+            class="author"
+            href="{{- (printf "authors/%s/" $uid) | absURL -}}"
+            rel="author"
+          >
+            {{- partial "core/get_author_image.html" (dict "author" . ) -}}
+            <span class="usa-sr-only">Posts by</span>
+            <span>{{- $displayName | default $uid | markdownify -}}</span> </a
+          >{{ if lt (add $i 1) $authors_count }}<!--,-->{{ end }}
 
+        {{ else }}
+          <div class="author grid-row">
+            <div class="grid-col-auto">
+              <div class="photo">
+                {{- partial "core/get_author_image.html" (dict "author" . ) -}}
+              </div>
+            </div>
+            <div class="grid-col-9">
+              <div class="details">
+                <span class="usa-sr-only">
+                  Originally posted by
+                  {{ $displayName | default $uid }}
+                  on
+                  {{ $.Date.Format "Jan 2, 2006" }}
+                </span>
+                <a
+                  href="{{ (printf "authors/%s/" $uid) | absURL }} "
+                  title="Posts by {{ $displayName | default $uid }}"
+                  rel="author"
+                  aria-label="Read more articles by {{ $displayName | default $uid }}"
+                >
+                  {{- $displayName | default $uid -}}
+                </a>
+                {{- if $agency -}}
+                  <p>
+                    {{- $agency -}}
+                  </p>
+                {{- end -}}
+              </div>
```

Comment:
There isn't much difference from Short and Long variants. Are there other use cases?

![image](https://github.com/GSA/digitalgov.gov/assets/3385219/154a4b2a-d5c1-4084-b05c-796c4d07d149)

Reply:
See the screenshot in the PR description above. The only difference is that `authors-list` displays as a vertical stacked list along with some additional spacing and layout style changes. See https://digital.gov/event/2022/07/20/white-house-and-gsa-open-innovation-forum-building-equitable-partnerships/ for a better example.

Reply:
Is there enough difference in the code to merit its own components? For example `authors-stacked`?